client/components/tinymce/iframe.scss

```diff
@@ -1,8 +1,4 @@
 body {
```

Comment:
Not sure which part we should keep or replace. There are some WordPress and TinyMCE specific styling that Foxlight doesn't' include.

Maybe we should remove typography(font size, line height, font family) and color related bits first?